types/office-js/index.d.ts

```diff
@@ -87181,13 +87182,27 @@ declare namespace Word {
          * [Api set: WordApi 1.7]
          */
         readonly checkboxContentControl: Word.CheckboxContentControl;
+        /**
+         * Specifies the combo box-related data if the content control's type is 'ComboBox'. It's `null` otherwise.
```

Comment:
This reads somewhat redundantly. I recommend this:

```suggestion
         * Specifies the data of the content control when its type is 'ComboBox'. It's `null` otherwise.
```

Similarly to the other "NOUN-related" constructions in this PR.